.setup/install_system.sh

```diff
@@ -762,6 +772,41 @@ if [[ ${VAGRANT} == 1 ]]; then
     systemctl enable nullsmtpd
 fi
 
+# Setup preferred_name_logging
+echo -e "Setup preferred name logging."
+
+function check_exit_code {
+# $1 = process return code
+# $2 = name of process
+    if [[ $1 -ne 0 ]]; then
+        echo -e "Error setting up $2.  Aborting."
+        exit 1
+    else
+        echo -e "Successfully set up $2."
+    fi
+}
```

Comment:
this file uses `set -ev` at the top of it so any line that has a non-zero exit code is already going to cause it to crash and die, as well as print the line it failed on immediately before a big error message. If we really need this, do `set +e` around this chunk, but my thought here is that we don't and can remove this function and its usages.